packages/react/src/primitives/utils/useLayoutEffect.ts

```diff
@@ -8,8 +8,6 @@ import * as React from 'react';
  *
  * See: https://reactjs.org/docs/hooks-reference.html#uselayouteffect
  */
-const useLayoutEffect = Boolean(globalThis?.document)
```

Comment:
The `Boolean` here is redundant (this was picked up by [no-extra-boolean-cast](https://eslint.org/docs/latest/rules/no-extra-boolean-cast), want to make it clear that it was not just an opinion)